script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -676,63 +680,63 @@ That reminds me.
+On the road to the square, countless pieces of art have been erected. Looking closely, I realize that they're human-sized cacti. And then, looking even closer, I realize that they aren't actually cacti, but people.
 }
 [sha:faa7c1c88469e4eb03a26de4ef0969f385b8159d]{
 -- Page 615, Offset 35035.
 -- 彼らの表面は<茨|いばら>のようなものでグルグル巻きで、サボテンに見えたのも仕方がない。
-They were turned and twisted like thorns on their surface, so it was only natural that I mistook them for cacti.
+Their bodies are all twisted up like thorns, so it's only natural that I mistook them for cacti.
 }
 [sha:ce85f4ef241a88c330883e0e55170c6e4a1352dc]{
 -- Page 615, Offset 35036.
 -- そのサボテンには真っ赤な花が咲いていた。人間を苗床にして咲く吸血花。血液で咲く<深紅の薔薇|ルージュメイアン>。
-A deep red flower bloomed from these cacti. A blood-sucking flower that used humans as a nursery. A crimson rose that blooms through their blood, Rouge Mailland.
+A deep red flower bloomed from these cacti. A vampiric flower that uses humans as a seedbed. A flower that blossoms by sucking human blood, Rouge Mailland.
 }
 [sha:bff721c708f928c45d3fddb222a679cc46297f4f]{
 -- Page 616, Offset 35037.
 -- サボテンになった人たちは頭ぐらいしかもう人間ではなかったけど、とても幸せそうな顔で笑っている。
-The humans who turned into cacti are no longer human apart from their faces, and yet they smile like they were in total bliss.
+Even though the only part of the cacti that still remains human is their heads, they're smiling in total bliss.
 }
 [sha:7eb279bb9374eb6f077301f1f1cca361bd25b458]{
 -- Page 616, Offset 35038.
 -- 死体に襲われて食べられたり、
-Although they were attacked and eaten by the Corpses,
+Although they were attacked and preyed upon by the corpses―――
 }
 [sha:4638955beed36f9498d4dfb9dcffb5f822cee585]{
 -- Page 616, Offset 35039.
 -- 死体と間違われて隣人に撲殺されたり銃殺されたり、
-and beaten and shot by their neighbors who mistook them for one,
+and beaten and shot by their peers who mistook them for enemies―――
 }
 [sha:d4a854e82e61863f767214639c14e84476f32eb9]{
 -- Page 616, Offset 35040.
 -- 壁に挟まれてジュースになったりするよりは上等な末路だと思った。だって夢見るような死に様で、わたしも殺されるのならこの方法がいいと思ったぐらい。
-in my opinion, their ends were better compared to the ones squeezed between the walls and turned into juice. After all, it was such a dream-like way of dying that if I were to be killed, I wouldn't mind such a way of death.
+I'd say they died more fulfilling deaths than the people who were crushed between the walls and turned into juice. I mean, they died in so much ecstasy that if I were to be killed too, I wouldn't mind going out like that.
 }
 [sha:58c40d9748b5b1789779013920995bc4f2a89db6]{
 -- Page 617, Offset 35041.
 -- たとえば一面の舞踏会。
-Meanwhile on the other side, there is a ball.
+Meanwhile, on the other side, there's a ball.
 }
 [sha:a6ed073c09aca7123fc3d6d81f543051383edc7f]{
 -- Page 617, Offset 35042.
 -- 道をずりずりと移動していたのは、大きな大きな虫たちだった。
-The thing that were squirming across the road were overly massive insects.
+Ginormous insects are squirming across the road.
```

Comment:
ummm spiders aren't insects...... they're arachnids........ taxonomically should be "arthropods" but i might just go for "bugs" even though people even more pedantic about the little fellas than i am might point out that bugs are a subset of insects, not a superset but WHATEVER

Reply:
yes captain